docs/bugs_by_version.json

```
@@ -1333,6 +1333,7 @@
     },
     "0.6.10": {
         "bugs": [
+            "DataLocationChange",
```

This sounds very generic. Maybe it would be better to make it more specific?
```suggestion
            "DataLocationChangeInInternalOverride",
```